dev/com.ibm.ws.webcontainer.servlet.4.0_fat.2/fat/src/com/ibm/ws/fat/wc/tests/WCRequestAutoDecompressTest.java

```diff
@@ -170,6 +200,13 @@ else if (compressMethod.equals("deflate"))
             }
             stream.close();
             byte[] compressedData = baos.toByteArray();
+            if(testCorruptedZip){
+                 LOG.info("Corrupting compressedData");
+                 System.out.println(compressedData);
+                 compressedData[compressedData.length-1] = '9'; //corrupt the last character
+                  System.out.println(compressedData);
```

Comment:
Instead of print statements, should we be using LOG to log this?

Reply:
Oops. Meant for debug -- I'll remove it.